pkg/limits/frontend/config.go

```diff
@@ -3,14 +3,20 @@ package frontend
 import (
 	"flag"
 	"fmt"
+
+	"github.com/grafana/dskit/ring"
+
+	util_log "github.com/grafana/loki/v3/pkg/util/log"
 )
 
 type Config struct {
-	ClientConfig BackendClientConfig `yaml:"client_config"`
+	ClientConfig     BackendClientConfig   `yaml:"client_config"`
+	LifecyclerConfig ring.LifecyclerConfig `yaml:"lifecycler,omitempty"`
 }
 
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	cfg.ClientConfig.RegisterFlagsWithPrefix("ingest-limits-frontend", f)
+	cfg.LifecyclerConfig.RegisterFlagsWithPrefix("ingest-limits-frontend.", f, util_log.Logger)
```

Comment:
Is the dot at the end of the prefix required?

Reply:
Yes I think so, it's not added otherwise https://github.com/grafana/dskit/blob/main/ring/lifecycler.go#L83-L90